src/endpoint/endpoint.js

```diff
@@ -36,11 +36,14 @@ const server_rpc = require('../server/server_rpc');
 const debug_config = require('../util/debug_config');
 const auth_server = require('../server/common_services/auth_server');
 const system_store = require('../server/system_services/system_store');
-const prom_reporting = require('../server/analytic_services/prometheus_reporting');
 const background_scheduler = require('../util/background_scheduler').get_instance();
 const endpoint_stats_collector = require('../sdk/endpoint_stats_collector');
 const { NamespaceMonitor } = require('../server/bg_services/namespace_monitor');
 const { SemaphoreMonitor } = require('../server/bg_services/semaphore_monitor');
+const prom_reporting = require('../server/analytic_services/prometheus_reporting');
+const cluster = /** @type {import('node:cluster').Cluster} */ (
+    /** @type {unknown} */ (require('node:cluster'))
+);
```

Comment:
do we need to uncomment line 47 in bucketspace? 
//if (this.stats) fs_context.report_fs_stats = this.stats.update_fs_stats;

Reply:
I didn't get you.